app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -180,9 +179,7 @@ const FundingAndFeeStep = (props: Props) => {
                     </OutcomesTD>
                     <OutcomesTD textAlign="right">{outcome.probability}%</OutcomesTD>
                     <OutcomesTD textAlign="right">
-                      <TDFlexDiv textAlign="right">
-                        0 <NewValue outcomeIndex={index} value="2.35" />
-                      </TDFlexDiv>
+                      <TDFlexDiv textAlign="right">0</TDFlexDiv>
```

Comment:
This is not part of the issue, we're not showing the price change in fee and funding step, so I removed it (anyways `NewValue` was returning null)